server/routes/index.ts

```diff
@@ -6,5 +6,5 @@ export default eventHandler(async (event) => {
   if (userExist === null) {
     throw createError({ message: "User not exists!", status: 409 });
   }
-  return userExist;
+  return omit(userExist.toObject(), ["privateKey"]);
```

Comment:
Use flattenMaps, because meta will not be transformed to object
```suggestion
  return omit(userExist.toObject({ flattenMaps: true }), ["privateKey"]);
```